apple/Sources/FerrostarMapLibreUI/Views/LandscapeNavigationView.swift

```diff
@@ -48,7 +50,10 @@ public struct LandscapeNavigationView: View, CustomizableNavigatingInnerGridView
         navigationState: NavigationState?,
         minimumSafeAreaInsets: EdgeInsets = EdgeInsets(top: 16, leading: 16, bottom: 16, trailing: 16),
         onTapExit: (() -> Void)? = nil,
-        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] }
+        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] },
+        @ViewBuilder currentRoadNameViewBuilder: @escaping (String?) -> AnyView = { name in
+            AnyView(CurrentRoadNameView(currentRoadName: name, theme: DefaultRoadNameViewTheme()))
+        }
```

Comment:
Duplicate extension from `Dynamically...`

Reply:
Let's discuss this and the next few comments on a call. I think we still haven't figured out the best approach but we're getting closer 😅

Reply:
Discussed on the call; can probably move to an extension defined on a protocol and hide the defaults inside the struct/init. The ViewModifier might need to be the only place that holds the closure reference.